setup.py

```diff
@@ -43,6 +43,7 @@ def build_extension(self, ext):
             "-DBUILD_UNIT_TESTS:BOOL=OFF",
             "-DMVDTool_VERSION=" + self.distribution.get_version(),
             "-DPYTHON_EXECUTABLE=" + sys.executable,
+            "-DEXTLIB_FROM_SUBMODULES:BOOL=ON",
```

Comment:
Could we turn that off by default, please?

Reply:
You're right. I forgot that `spack` uses this for `py-mvdtool`.

Reply:
Actually, I think that this can be left as `ON` by default in`setup.py`.
`EXTLIB_FROM_SUBMODULES=ON` simply allows to install the projects from the submodules if they are not found in the environment  according to the following:
https://github.com/BlueBrain/MVDTool/blob/d388814f00d97c8f0ddfd61fd5f2f58aa8391031/CMakeLists.txt#L53
In case that `MVDTool` is installed by `setup.py` with loaded the proper modules or `spack`, `highfive` and `libsonata` should be already found because they are found in the environment, so the submodule won't be used.
The problem with this `OFF` as default in the `setup.py` is that somehow then this must be passed as argument to `pip` or somewhere else to use the submodules if the related packages are not installed and I didn't find how this is possible.
What do you think about that?

Reply:
Ah sorry, I think I got a bit confused.

I guess this looks acceptable, although I still don't really like these changes, I think it should be up to the user to provide the proper dependencies.

Reply:
I agree that it's a bit unclear way to do this. I don't think there is a better way to do this though because of the lack of python packages for `HighFive`. To be safe in `spack` we could maybe add a patch which disables the `EXTLIB_FROM_SUBMODULES` variable in `setup.py`.